docs/pages/guides/environment-variables.mdx

```diff
@@ -45,7 +46,13 @@ When you run `npx expo start`, `process.env.EXPO_PUBLIC_API_URL` will be replace
 
 Expo CLI loads **.env** files according to the [standard .env file resolution](https://github.com/bkeepers/dotenv/blob/c6e583a/README.md#what-other-env-files-can-i-use) and then replaces all references in your code to `process.env.EXPO_PUBLIC_[VARNAME]` with the corresponding value set in the **.env** files. Code inside **node_modules** is not affected for security purposes.
 
-Every variable must be referenced as a static property in order for it to be inlined. For example, the expression `process.env.EXPO_PUBLIC_KEY` will be rewritten, but `process.env['EXPO_PUBLIC_KEY']` will not.
+### How to read from environment variables
+
+- <YesIcon small /> **Every environment variable must be referenced as a static property for it to be
```

Comment:
If possible, can a link be included explaining what a `static property` is referring to. I feel like this term isn't generally understood. If there is a good link that explains it that would be great to include here.

Reply:
I didn't find a direct link that explains the context of the environment variable as a static property but did add some context (see line 51) to explain what it means to be a static property in this context.

Reply:
I'm good with this, someone can resolve this conversation as I cannot resolve it.

Reply:
There are two different definitions of "static properties" here. The original documentation (not line 51) is talking about the property name being static. The extra clarification on line 51 is talking about the property value being static.

Furthermore, these values aren't runtime environment variables and they aren't loaded into the app's process. They are the build computer's environment variables and are inlined into the code. Will add a suggestion to this PR.